container_registry/container_analysis/create_occurrence.go

```diff
@@ -26,7 +26,10 @@ import (
 
 // createsOccurrence creates and returns a new Occurrence of a previously created vulnerability Note.
 func createOccurrence(resourceURL, noteID, occProjectID, noteProjectID string) (*grafeaspb.Occurrence, error) {
-	// resourceURL := fmt.Sprintf("https://gcr.io/my-project/my-image")
+   	// Use this style of URL when you use Google Container Registry
+	// resourceURL := fmt.Sprintf("https://gcr.io/my-project/my-repo/my-image")
+	// Use this style of URL when you use Google Artifact Registry
+	// resourceURL := fmt.Sprintf("https://LOCATION-docker.pkg.dev/my-project/my-repo/my-image")
```

Comment:
No need for the `Sprintf`. Can just do a string directly.
```suggestion
	// resourceURL := "https://gcr.io/my-project/my-repo/my-image"
	// Use this style of URL when you use Google Artifact Registry
	// resourceURL := "https://LOCATION-docker.pkg.dev/my-project/my-repo/my-image"
```